### PR TITLE
ISSUE-1: Configure registration screen

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: openmrs-configs/configuration/concepts/disposition.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Answers,Members,_version:1,_order:500
+efaf4399-601c-4479-aba2-21591a6eccae,,org.openmrs.module.emrapi:TRANSFER,Transfer Patient,Transfer Patient,Transfer Patient,Transfer Patient,,,Misc,N/A,,,,
```
Comment (reagan-meant) [inline openmrs-configs/configuration/concepts/disposition.csv]: Do we need this file?
Doesn't the disposition work with out it?

File: openmrs-configs/configuration/patientidentifiertypes/patientidentifiertypes.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Uuid,Void/Retire,Name,display:en,Description,Required,Format,Format description,Validator,Location behavior,Uniqueness behavior
+a71403f3-8584-4289-ab41-2b4e5570bd45,,SSN,SSN,Social Security Number,FALSE,[A-Z]{1}-[0-9]{7},"Identifier should be 1 letter, followed by a dash and 7 numerical characters. Eg, A-0010902",,NOT_USED,UNIQUE
+aeccdb8e-0766-4150-98cd-9205bdd575d5,,National ID,National ID,National ID Number,FALSE,^[0-9]{9}([(][0-9]{2}[)])?$,"National ID should be 9 digits. E.g, 923456678. If the ID is renewed, it will have (xx) appended at the end, where xx represent the iteration of card renewed e.g (01) for first renewal, (02) for second renewal, and so on.",,NOT_USED,UNIQUE
```
Comment (reagan-meant) [inline openmrs-configs/configuration/patientidentifiertypes/patientidentifiertypes.csv:2]: Do we need all these IDs?

File: openmrs-configs/configuration/roles/roles.csv [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
+4a899553-f1f6-4c5a-8647-09572d5d9ea8,Doctor,"User have ability to login and access registration and clinical modules and perform registration and clinical operations(search patient, register patient, see the search results in the clinical queues)",Provider;FrontDesk;Registration-App;Programs-App;Clinical-App;Radiology-App;PatientDocuments-App;Reports-App;Admin-App;OrderFulfillment-App;Appointments:FullAccess,app:clinical;Get Care Settings;Get Patient Programs;Get Beds;Get Order Frequencies;Get Admission Locations;View Program Attribute Types;app:admin;Get conditions;Get Diagnoses;app:clinical:observationTab;app:clinical:diagnosisTab;app:clinical:consultationTab;app:clinical:ordersTab;app:clinical:treatmentTab;Add Orders;Edit Orders;app:reports;Delete Patient Document;Edit Conditions;Send Email;app:document-upload;Get Bed Type;Edit Bed Type;Get Bed Tags;Edit Bed Tags;Edit Beds;Manage Locations;Get Location Attribute Types;Manage Location Attribute Types;Manage Location Tags
+20e3cb77-1b5c-4ec7-9a12-aca4d3d61ad2,Admin,,SuperAdmin,
```
Comment (reagan-meant) [inline openmrs-configs/configuration/roles/roles.csv]: Am reverting this for now.
I will adjust the roles later

File: openmrs-configs/configuration/concepts/disposition.csv (hunk 1 above)
Comment (reagan-meant) [reply openmrs-configs/configuration/concepts/disposition.csv]: Am reverting for now

**Review by reagan-meant — COMMENTED** [review]
Removed unneccessary files

File: openmrs-configs/configuration/concepts/disposition.csv (hunk 1 above)
Comment (WodPachua) [reply openmrs-configs/configuration/concepts/disposition.csv]: Alright, thanks

**Pull request merged**